_i18n/en/general/privacy.md

```diff
@@ -1,58 +1,101 @@
-- Google: If you have activated backup & reset in your phone settings (Settings → Backup & Reset → Back up my data), you should be aware that Android itself will periodically save a copy of your phone's data in Google's servers. This backup contains private information, including your WiFi passwords, messages and call history. It may also include data from AntennaPod and from other apps you use. The developers of AntennaPod do not have access to this data. For more information, see [Google's privacy policy](https://policies.google.com).
+**Except for third-party services, all data provided and created when using the AntennaPod app is stored locally on your device.
+This means your data is not sent or uploaded anywhere, except when the you search, or turn on third-party services in the app.**
+When interfacing with third-party services, the app itself receives subscriptions and playback events provided by the respective services.
 
-## What data might be collected, stored and processed from AntennaPod online services
-- The donation system: AntennaPod uses Open Collective, offered by Open Collective Inc (USA) & Open Collective Europe ASBL (Belgium). When you make a donation, these entities get access to certain data. For more information, see [Open Collective's privacy policy](https://opencollective.com/privacypolicy).
-- The website: AntennaPod's website is hosted via GitHub Pages. [GitHub's privacy policy](https://docs.github.com/en/github/site-policy/github-privacy-statement#github-pages) notices that "GitHub may collect User Personal Information from visitors to your GitHub Pages website, including logs of visitor IP addresses, to comply with legal obligations, and to maintain the security and integrity of the Website and the Service." The website does not store any cookies on your machine, and no 3rd party tracking (analytics) or other services are used.
-- The forum: AntennaPod's forum uses the Discourse software, hosted by the core team. Please see the [forum's privacy policy](https://forum.antennapod.org/privacy) for which data is concerned.
+## App data third-parties collect, store and process
 
-## What data the AntennaPod core team may have access to
-The developers of AntennaPod do **not** have access to any of your information, except when you
+- Discovery and search services: When opening the Discover screen in the app, it sends a call to Apple to collect podcast suggestions.
+They may store the query, including country (which by default is the device's country).
+These suggestions can be disabled by choosing "Off" in the region selector.
+When using the search feature in AntennaPod, the services PodcastIndex.org, iTunes, fyyd, gpodder.net, mzstatic.com, and podnews.net, and omnycontent.com
```

Comment:
```suggestion
When using the search feature in AntennaPod, the services PodcastIndex.org, iTunes, fyyd, and gpodder.net
```
Don't really know what the ones I added do, but they do (try to) connect when searching. I didn't try it on a clean install, so it might be from podcasts I added.